### DISCUSSION TITLE
Does SMPlayer have any specific support for audiobooks/multiple-file compilations?

### DISCUSSION THREAD

**Discussion by JoanEliot** [smplayer-dev/smplayer#169] [Q&A]
Does SMPlayer support audiobooks, as such? If not, what tools does it have among those needed for a good audiobook listening experience? These include:

1. _Per-album_ resume from last play position. Can you play an audio album rather than a track, and when you do, can SMPlayer be set to pick up from the track and the play position in that album that was last played?
2. Bookmarks: SMPlayer has bookmarks, right? Can they be viewed per album rather than per track?
3. I don't suppose you support text annotations...

**Comment by JonnJonzz** [comment]
SMPlayer works quite well for M4B files. It also works with MP3 files (either standalone or within a ZIP file), though it has its limitations (see my next paragraph below). Playback from last position is supported, though it's most useful for M4B files, since those are normally many hours long for each file. For example, an audiobook that's 12 hours long may contain two 6 hour M4B files, so you just need to remember which half of the book you are currently listening to.

For MP3 files (which are normally split up into many smaller files), in order to resume your listening, you'll need to remember which was the last file you were playing, and SMPlayer will remember your position within that file. Unfortunately, if you are listening to an MP3 file within a ZIP file, SMPlayer only seems to remember your position within the **first** file of that ZIP (for an audio-only program that does remember your last position of multiple MP3 files within a ZIP, check out the freeware program XMPlay (http://support.xmplay.com).
